lib/live_debugger/app/debugger/web/components/elixir_display.ex

```diff
@@ -66,7 +61,7 @@ defmodule LiveDebugger.App.Debugger.Web.Components.ElixirDisplay do
 
   attr(:items, :list, required: true)
 
-  defp text_items(assigns) do
+  def text_items(assigns) do
```

Comment:
```suggestion
  defp text_items(assigns) do
```

Does it have to be public?